bls-eth-wasm/bls.d.ts

```diff
@@ -69,6 +69,10 @@ declare class SignatureType extends Common {
     serializeUncompressed (): Uint8Array;
     deserializeUncompressedHexStr (s:string): void;
     serializeUncompressedToHexStr(): string;
+    isValidOrder(): boolean;
```

Comment:
I think `PublicKeyType` needs to be updated too?

Reply:
Oh yeah, nice catch